howto/install/deploy-juju.md

```diff
@@ -151,15 +151,15 @@ machines:
     constraints: "instance-type=m4.2xlarge root-disk=50G"
 ```
 
-To deploy with the bundle from above, add `--overlay overlay.yaml` to your deploy command:
+To deploy, add `--overlay overlay.yaml` to your deploy command. For example:
 
     juju deploy cs:~anbox-charmers/anbox-cloud ... --overlay overlay.yaml
```

Comment:
Does the "..." mean that you might need to add something? Probably the `--overlay ua.yaml`?
In this case, wouldn't it make more sense to just add this here, since most (all?) users will need it and this is only an example anyway?